survey-service/src/main/java/pl/hycom/surveyservice/controller/NewSurveyController.java

```diff
@@ -5,19 +5,19 @@
 import org.springframework.http.HttpRequest;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;

```

Comment:
Nie jest to błąd, lecz (nie do końca mój) faszyzm - wszelkiego rodzaju checkstyle itd będą się pluć, że używamy zbyt ogólnych importów.

Reply:
Ok, rozdzieliłem